WooCommerce/Classes/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModel.swift

```diff
@@ -692,6 +710,29 @@ extension DefaultProductFormTableViewModel {
                 return "\(trialLength) \(trialPeriod.descriptionPlural)"
             }
         }
+
+        // Subscription Expire After
+        static let subscriptionExpiryTitle = NSLocalizedString("defaultProductFormTableViewModel.expireAfter",
+                                                               value: "Expire after",
+                                                               comment: "Title for Subscription expiry row in the product form screen.")
+        static let neverExpire = NSLocalizedString("defaultProductFormTableViewModel.neverExpire",
+                                                   value: "Never expire",
```

Comment:
np: I understand this follows Core but I think "Never expires" is more grammatically correct (ref: https://oneminuteenglish.org/en/verb-tense-after-never/ "Never in present tense")

Reply:
Good point. Updated the text in 887a46459098d564bd418e024a73f3ad96d087d7